hooks/useSubgraphDeployment.ts

```diff
@@ -117,13 +124,20 @@ signature: ${signature}`
       options.node === 'hosted' ? '/api/graph/deploy' : 'https://api.studio.thegraph.com/deploy/'
 
     try {
-      await deployPreparedSubgraph(subgraph, deployState.files, {
+      const deployResult = await deployPreparedSubgraph(subgraph, deployState.files, {
         node,
         subgraphName: options.name,
         deployKey: options.accessToken,
       })
 
       setDeployState(_state => ({ ..._state, status: DeployStatus.DEPLOY_COMPLETE }))
+      update(_subgraph => ({
+        ..._subgraph,
+        publications: [
+          ..._subgraph.publications,
+          { version: deployResult.version, cid: deployResult.manifestCid },
```

Comment:
Can you add the `node` to the publications (might need to add `node` to the struct in a separate file too.

This could be useful for error checking, as only the Subgraph Studio cares about version numbers.

Reply:
I've added it to the publications. Which separate file are you referring to?

Reply:
Can you add it to the type definition here:

https://github.com/crypto-stats/site/blob/main/hooks/local-subgraphs.ts#L8-L11